### PR TITLE
CAM-14449: document admin diagnostics feature

### PR DESCRIPTION
**Pull request by cliftondavies**
related to CAM-14449

### CODE REVIEW

File: content/webapps/admin/system-management.md [hunk 1]
```diff
@@ -13,12 +13,18 @@ menu:
...
 
 {{< note title="Accessing the System Settings menu" class="info" >}}
 The System Settings menu is only usable by users which are granted with *All* permission for authorizations.
 {{< /note >}}
 
+# Diagnostics
+
+{{< img src="../img/admin-diagnostics.png" title="Diagnostics" >}}
+
+The Diagnostics menu allows you to view and copy telemetry data about your environment or distribution of Camunda. The aim is to improve problem diagnosis by ensuring our customer facing departments are able to quickly understand the Camunda environment that you are running.
```
Comment (tasso94) [inline content/webapps/admin/system-management.md:26]: Similar feedback that I already gave on the translation string. 

The primary product management motivation to invest in that topic is that our customer-facing departments can request this information from users. However, this shouldn't be so much of an interest for all of our users (especially not for our community users who don't have direct access to our support services). The feature might also help us do community work or in the forum.

Can you generalize more about how a user can benefit from this feature? I think we should especially remove the term `customer facing departments`. 

The backend documentation that already exists might help you as well: https://docs.camunda.org/manual/develop/reference/rest/telemetry/get-telemetry-data/.

**Review by tasso94 — CHANGES_REQUESTED** [review]
Please see my review hint below:

**Review by tasso94 — APPROVED** [review]
I really like how you incorporated my feedback! Reads very nice now. 🙂 👍 

Going forward please request a technical writing review from `@christinaausley`.

**Review by christinaausley — APPROVED** [review]
Bolded the section name in latest commit, as all UX components referenced are to be bolded. LGTM otherwise!

**Pull request merged**